app/next.config.js

```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
+  env: {
+    API_URL: process.env.API_URL,
+  }
```

Comment:
Is it possible to implement this for API to have process.env variable accessible anywhere without doing the confog() import

Reply:
I mean, class that is here just to stock process.env variables